changelog/README.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 7th May 2024
+* GA4 `purchase` event now [lists products and rooms as separate items](../integrations/google-triggers-reference.md#purchase), aligning with the presentation in all preceding events throughout the flow
```

Comment:
This is enough
```suggestion
* GA4 `purchase` event now [lists products and rooms as separate items](../integrations/google-triggers-reference.md#purchase)
```

Reply:
Or maybe I don't understeand fully the meaning?

Reply:
Nah, let’s remove it